safehttp/header_test.go

```diff
@@ -81,9 +83,11 @@ func TestSetImmutable(t *testing.T) {
 	}
 }
 
-func TestSetEmptyImmutable(t *testing.T) {
+func TestSetEmptyClaimed(t *testing.T) {
 	h := newHeader(http.Header{})
-	h.MarkImmutable("Foo-Key")
+	if _, err := h.Claim("Foo-Key"); err != nil {
+		t.Errorf(`h.Claim("Foo-Key") got: %v want: nil`, err)
+	}
 	if err := h.Set("Foo-Key", "Bar-Value"); err == nil {
 		t.Error(`h.Set("Foo-Key", "Bar-Value") got: nil want: error`)
```

Comment:
Idk which is the better option, but I wondering if it wouldn't be more helpful to output more info about what sort of error occurred.

Reply:
When this line runs, then no error has occurred. I could include more information about which error should have occurred, but if we in the future change the error message that is returned somehow then you would have to update this line as well and that is difficult to remember.